examples/torch_lightgbm.py

```diff
@@ -0,0 +1,331 @@
+from __future__ import annotations
+
+import os
+from abc import abstractmethod
+
+# =====================
+# LightGBM code - hacked to work with torch tensor inputs
```

Comment:
why not convert torch tensor to whatever works with the torch-frame implementation of GBDT?
doesn't look like this script should be public-facing.